lib/parser.js

```diff
@@ -62,7 +64,7 @@ function parseSceneLikeWithX(filename) {
   }
 }
 function parseSceneLike000(filename) {
-  var m, result = {}
+  var m, result = { episodeCount: 1 }
 
   if (m = filename.match(/^(.+?)([\. ])(?:((?:20|19)\d\d)\2)?(\d\d\d+)\2/i)) {
```

Comment:
Here I'm more uncertain if the common format would be `-dd` as well. Strange format anyway. =)

Reply:
yeah definitely not sure about this one haha
-dd seems safe though?